frontend/src/Pages/SettingDialog.js

```diff
@@ -22,7 +22,7 @@ const SettingDialog = () => {
                 </Button>
             </div>
         </div>
-    );
-};
+    </div>
+);
```

Comment:
could you please remove the additions and bring back the deletions. This is causing build errors.

Reply:
nevermind, I think the whole file should be reverted :)

Reply:
yeap ive reverted the whole file